MatrixSDK/Data/MXRoom.h

```diff
@@ -1304,6 +1304,23 @@ Remove a tag applied on an event of the room
  */
 - (void)acknowledgeEvent:(MXEvent*)event andUpdateReadMarker:(BOOL)updateReadMarker;
 
+/**
+ It will set the entire room as unread.
+ This is only local since it's not possible to remove from the server the read events.
+ */
+-(void)setUnread;
```

Comment:
Would be good to align with the `MXStore` api, either having `set/resetUnread` or `set/resetUnreadMarker` everywhere